packages/loading-scripts/src/index.ts

```diff
@@ -0,0 +1,72 @@
+/* eslint-disable no-new */
+/** Loading snippet */
+const sdkBaseUrl = 'https://cdn.rudderlabs.com/beta/v3';
+const sdkName = 'rudder-analytics.min.js';
+const writeKey = '<write-key>';
+const dataPlaneUrl = '<data-plane-url>';
+const options = {
+  // configure your load options here
+};
```

Comment:
add a variable called disableDynamicImports with default value false. If the consumer set it to true never retrieve the modern package (ignore the detection try catch) in case someone does not want the extra requests

Reply:
Adressed